src/Workspaces/MSBuildTest/Utilities/DotNetSdkMSBuildInstalled.cs

```diff
@@ -117,10 +117,8 @@ public DotNetSdkMSBuildInstalled()
         {
         }
 
-        // Eventually should revert condition to 'SdkPath is null' after MSBuild issue is fixed:
-        // https://github.com/dotnet/roslyn/issues/67566
         public override bool ShouldSkip
-            => true;
+            => SdkPath is null;
```

Comment:
Should we be closing the bug too?

Reply:
It is called out in the description as being resolved by this PR.

Reply:
No longer applicable